Cargo.toml

```diff
@@ -50,6 +50,7 @@ members = [
   "oak_virtio",
   "sev_serial",
   "snp_measurement",
+  "testing/hello_world_linux_init",
```

Comment:
Will this hello world always be the initial process from which everything is forked?
Or this one is just for testing/experimenting?

Reply:
This is just for initial testing and experimenting.